CMakeLists.txt

```diff
@@ -102,7 +102,7 @@ else ()
     )
 endif()
 
-set(CHI_LIBS lua m dl ${MPI_CXX_LIBRARIES} petsc ${VTK_LIBRARIES})
+set(CHI_LIBS stdc++ lua m dl ${MPI_CXX_LIBRARIES} petsc ${VTK_LIBRARIES})
```

Comment:
I threw this in here because it seems some compiler configurations on TACC wants this to be there. It should be harmless.

Reply:
Can you include that modification in ```Downstream.cmake``` as well?

Reply:
Done